aws/ec2/ec2/vm.go

```diff
@@ -54,8 +54,8 @@ func NewVM(ctx *pulumi.Context) (*VM, error) {
 	return vm, nil
 }
 
-func NewDefaultEC2Instance(e aws.Environment, name, instanceType string) (*ec2.Instance, remote.ConnectionOutput, error) {
-	awsInstance, err := NewEC2Instance(e, name, "", AMD64Arch, instanceType, e.DefaultKeyPairName(), "", "default")
+func NewDefaultEC2Instance(e aws.Environment, name, arch, instanceType string) (*ec2.Instance, remote.ConnectionOutput, error) {
```

Comment:
@pducolin @vboulineau 
Made a small change to allow users to specify arch of EC2 instance. Any issues with this?

Reply:
Why not using `NewEC2Instance` to define the architecture instead of `Default` Wold keep `default` with as low params as possible

Reply:
Makes sense!